webapp/lib/platform.dart

```diff
@@ -7,7 +7,7 @@ import 'platform_constants.dart' as platform_constants;
 
 Logger log = new Logger('platform.dart');
 
-firestore.Firestore _firestoreInstance;
+firestore.Firestore firestoreInstance;
```

Comment:
is the switch from private to public intentional? I think in Nook we wrapped the firestore in a set of helper methods, I think we want to do the same here long term, so no reason for it to be public?

Reply:
Oh I see.

I was working on two things at the same time - switching over to the type, and integrating the type with Firebase - this change was part of the second package that seemingly I didn't fully back out before sending the PR.

If you're ok with it, in order to continue sequencing the series of changes, I'll leave this switch to public in, and will switch back to private when we land the wrapper?

Reply:
sounds good thanks!